apps/content/tests/internal/test_recitation_list.py

```
@@ -184,12 +184,26 @@ def test_list_recitations_where_fetching_recitations_should_return_response_with
     # ── Query count ────────────────────────────────────────────
 
     def test_list_recitations_should_use_constant_number_of_queries(self):
+        baker.make(
+            Asset,
+            _quantity=20,
+            category=Resource.CategoryChoice.RECITATION,
```

Ruff (0.15.10)

[error] Undefined name `Resource`

(F821)